app/assets/javascripts/app/radio-btn.js

```diff
@@ -0,0 +1,39 @@
+import 'classlist.js';
+
+function clearRadioBtn(name) {
+  const radioGroup = document.querySelectorAll(`input[name='${name}']`);
+
+  for (let i = 0; i < radioGroup.length; i++) {
+    radioGroup[i].parentNode.parentNode.classList.remove('bg-light-blue');
+  }
+}
+
+function radioBtn() {
+  const radios = document.querySelectorAll('.radio-btn input[type=radio]');
+
+  if (radios) {
+    for (let i = 0; i < radios.length; i++) {
+      const radio = radios[i];
+      const label = radio.parentNode.parentNode;
+      const name = radio.getAttribute('name');
+
+      if (radio.checked) label.classList.add('bg-light-blue');
+
+      radio.addEventListener('change', function() {
+        clearRadioBtn(name);
```

Comment:
CodeClimate is reporting a warning "Don't make function within a loop" — and I think it's right here, the values of`name` and `radio` that get captured will probably change for each radio, we need to use an IIFE to capture the values immediately.

```js
radio.addEventListener('change', ((name, radio) =>
  (name, radio) => {
    clearRadioBtn(name);
    if (radio.checked) label.classList.add('bg-light-blue')
  })(name, radio))
```